index.js

```diff
@@ -289,8 +290,8 @@ function choosePluralForm(text, locale, count) {
 // interpolation placeholder.
 var dollarRegex = /\$/g;
 var dollarBillsYall = '$$';
-var tokenRegex = /%\{(.*?)\}/g;
-function interpolate(phrase, options) {
+function interpolate(phrase, options, tokenPattern) {
+  var tokenRegex = new RegExp(tokenPattern, 'g');
```

Comment:
Hi @ljharb! 
You mean parse the input phrase? 

And, what do you think that may happen if we allow that? I don't see a problem. 

Thanks!
Luciano